libdrgn/orc_info.c

```diff
@@ -21,6 +21,8 @@
 
 DEFINE_VECTOR(uint64_range_vector, struct uint64_range);
 
+#define ORC_HASH_SIZE 20
```

Comment:
If I may bikeshed, how about `ORC_HEADER_SIZE`? (Since it's an implementation detail that the header is a hash. Clearly I didn't follow that logic in my own naming of `orc_hash_6_4` and co.)